stable/mlrun-kit/templates/mlrun-persistency/secret.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if eq .Values.mlrun.storage "s3" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "mlrun-kit.mlrun.api.fullname" . }}-s3-secret
```

Comment:
```suggestion
  name: {{ template "mlrun-kit.mlrun.api.fullname" . }}-s3-auth
```


this is a secret, no need to specify that. Also would be better to template that (_helpers.tmpl)